### ISSUE TITLE
lightly/lightly/loss /pmsn_loss.py  The kl divergence calculation loss is negative.

### ISSUE THREAD

**Issue by yfchen1997** [lightly-ai/lightly#1194]
Dear Contributor,
When i use lightly/lightly/loss /pmsn_loss.py to calculate regularization loss with a power law target distribution. I find the loss = F.kl_div(input=mean_anchor_probs, target=power_dist, reduction="sum") < 0 (tensor(-6.8904, device='cuda:0', grad_fn=<KlDivBackward0>)).
Thank you for your help in clarifying this.

**Comment by guarin** [comment]
Just saw this now, answered in #1058

**Comment by guarin** [comment]
Closing this in favor of #1058
